public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/form.php

```diff
@@ -1,16 +1,29 @@
 <?php
 
 namespace WordCamp\SpeakerFeedback\Form;
+use const WordCamp\SpeakerFeedback\QUERY_VAR;
 use function WordCamp\SpeakerFeedback\{ get_views_path, get_assets_url };
 
 add_filter( 'the_content', __NAMESPACE__ . '\render' );
 add_filter( 'wp_enqueue_scripts', __NAMESPACE__ . '\enqueue_assets' );
 
+/**
+ * Check if the current page is the feedback form.
+ */
+function is_feedback_page() {
```

Comment:
The name of this function is confusing since it actually is for a different purpose than `is_page( get_option( 'feedback_page' ) )`. Maybe something like `wants_feedback_form()`?

Reply:
Yeah, I went back and forth a bit on it. Maybe `is_session_feedback_page()`? I try to stick to `is_`/`has_` names when a function returns a boolean.

Reply:
I think `is_session_feedback_page()` has the same problem, that it sounds like it is referring to the actual page, rather than a single session that includes the feedback form.

`has_feedback_form()`?

Reply:
> that it sounds like it is referring to the actual page

It is referring to the feedback form page though, with this approach there is no "main" feedback form, just the navigation dropdown from #357  & the feedback forms on the session pages.

Reply:
By "actual page" I mean `site.worcamp.org/feedback/` where the dropdown is.

I think I'm getting tripped up by having "page" be in the name here, since that means something very specific in the WP context.

All the function is actually doing is checking for the existence of a particular query parameter, right? And the intention the query parameter is signaling is that we want to append a feedback form to the current view. In the future, that might even be for other post types besides sessions (e.g. `workshop` or something). That's why I think it would be best to use something like `has_feedback_form` or `wants_feedback_form`.

Reply:
Good point about other session types. I've updated it to `has_feedback_form`.